tests/e2e/main.go

```diff
@@ -430,13 +442,13 @@ TEST RESULTS
 		}
 	}
 	if len(passedTests) > 0 {
-		report += fmt.Sprintln("\n\nPASSED TESTS:\n")
```

Comment:
Removed new lines because of:
```
cd tests/e2e; go test -timeout 30s -run ^TestWriteExamples -v
# github.com/cosmos/interchain-security/v4/tests/e2e
./main.go:445:13: fmt.Sprintln arg list ends with redundant newline
./main.go:451:13: fmt.Sprintln arg list ends with redundant newline
```